components/team_members_dropdown/team_members_dropdown.test.tsx

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import TeamMembersDropdown from 'components/team_members_dropdown/team_members_dropdown';
+
+describe('components/team_members_dropdown', () => {
+    const user = {
```

Comment:
We have a mock you can use here instead of having to mock it yourself.
Take a look at `utils/test_helper.ts`